src/status_im/subs/chats.cljs

```diff
@@ -123,35 +123,38 @@
  :<- [:contacts/blocked-set]
  :<- [:contacts/contacts-raw]
  :<- [:chat/inputs]
- (fn [[{:keys [group-chat chat-id chat-name name] :as current-chat} my-public-key community
+ (fn [[{:keys [group-chat chat-id chat-name name] :as current-chat}
+       my-public-key
+       community
        blocked-users-set contacts
        inputs]]
    (when current-chat
-     (cond-> current-chat
-
-       (and (chat.events/community-chat? current-chat)
-            (get-in community [:chats (subs (:chat-id current-chat) 68) :can-post?]))
-       (assoc :able-to-send-message? true)
-
-
-       (and (chat.events/group-chat? current-chat)
-            (member? my-public-key current-chat))
-       (assoc :able-to-send-message? true
-              :member?               true)
-
-       (not chat-name)
-       (assoc :chat-name name)
-
-       (not group-chat)
-       (assoc
-        :contact-request-state (get-in contacts [chat-id :contact-request-state])
-        :able-to-send-message?
-        (and
-         (or
-          (get-in inputs [chat-id :metadata :sending-contact-request])
-          (= constants/contact-request-state-mutual
-             (get-in contacts [chat-id :contact-request-state])))
-         (not (contains? blocked-users-set chat-id))))))))
+     ;; 68 Is the length of the community-id prefix in the `chat-id`
+     (let [relative-chat-id (-> current-chat :chat-id (subs 68))]
```

Comment:
@ulisesmac this logic is already present in the function `data-store/community-chat-id->channel-id`.